docs/source/sparse.rst

```diff
@@ -300,7 +300,7 @@ To use these ops, simply pass the output of ``to_sparse_semi_structured(tensor)`
     >>> torch.allclose(c, torch.mm(a_sparse, b))
     True
 
-Under the hood, SparseSemiStructuredTensor will call ``torch._structured_sparse_linear`` for accelerated inference using CUTLASS sparse kernels.
```

Comment:
We can remove this entirely. We don't document private functions.

Reply:
Fixed.